src/store/model-store.js

```diff
@@ -1,33 +1,26 @@
 import { Store } from 'consus-core/flux';
-
+import { searchModel } from '../lib/api-client';
 let model = null;
 let models = [];
 
 class ModelStore extends Store {
 
     getModel() {
-        if (model === null) {
-            return model;
-        }
-        return {
-            address: model.address,
-            name: model.name
-        };
+        return model;
```

Comment:
Oh...I see the problem now. What if I returned something like `JSON.parse(JSON.stringify(model));`?